app/src/main/java/cm/aptoide/pt/app/view/MoreBundlePresenter.java

```diff
@@ -151,11 +162,17 @@ private void handleError(HomeBundlesModel.Error error) {
     view.getLifecycleEvent()
         .filter(lifecycleEvent -> lifecycleEvent.equals(View.LifecycleEvent.CREATE))
         .flatMap(created -> view.moreClicked()
-            .doOnNext(homeMoreClick -> homeAnalytics.sendTapOnMoreInteractEvent(
-                homeMoreClick.getBundlePosition(), homeMoreClick.getBundle()
-                    .getTag(), homeMoreClick.getBundle()
-                    .getContent()
-                    .size()))
+            .doOnNext(homeMoreClick -> {
+              homeAnalytics.sendTapOnMoreInteractEvent(homeMoreClick.getBundlePosition(),
+                  homeMoreClick.getBundle()
+                      .getTag(), homeMoreClick.getBundle()
+                      .getContent()
+                      .size(), chipManager.getCurrentChip());
+              if (chipManager.getCurrentChip() != null) {
+                homeAnalytics.sendChipTapOnMore(homeMoreClick.getBundle()
+                    .getTag(), chipManager.getCurrentChip());
```

Comment:
Same as above, since we are already checking if it is not null, we could just send the name which is what is needed.